server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -71,6 +73,7 @@ public class ExtensionsOrchestrator implements ReportingService<PluginsAndModule
     public static final String REQUEST_EXTENSION_REGISTER_REST_ACTIONS = "internal:discovery/registerrestactions";
     public static final String REQUEST_OPENSEARCH_NAMED_WRITEABLE_REGISTRY = "internal:discovery/namedwriteableregistry";
     public static final String REQUEST_OPENSEARCH_PARSE_NAMED_WRITEABLE = "internal:discovery/parsenamedwriteable";
+    public static final String REQUEST_REST_EXECUTE_ON_EXTENSION_ACTION = "internal:extensions/restexecuteonextensiontaction";
```

Comment:
If we have the action named `internal:extensions` in place, can we use the same for the above written actions as well? Just to maintain the uniformity around all the ExtensionOrchestrator actions.

Reply:
I think these are based on the package name where the Action resides.  So we could change it and move the action to another package if we wanted.

Reply:
We could define one for extensions, if you'd like to have a dedicated namespace. 
`TransportService` validates these action names[1]

[1] https://github.com/opensearch-project/OpenSearch/blob/d377ab13c9205cc94c52042c15cad6c17fb0df1e/server/src/main/java/org/opensearch/transport/TransportService.java#L1057

Reply:
Sounds like a good proposal, @saratvemulapalli, suggest turning your comment into an issue.

Reply:
https://github.com/opensearch-project/OpenSearch/issues/4334